pages/_app/index.jsx

```diff
@@ -14,6 +17,9 @@ import { initStore, GlobalProvider } from 'store'
 import Application from 'components/application'
 import { Sentry } from 'utils/sentry'
 
+dayjs.extend(relativeTime)
+dayjs.extend(advancedFormat)
```

Comment:
What do these plugins do?

Can we please get rid out from `dayjs` in favour of `Intl.DateFormat`?

Reply:
Once issues are merged we can.